pkg/kubeapiserver/server/insecure_handler.go

```diff
@@ -63,7 +63,7 @@ type InsecureServingInfo struct {
 	BindNetwork string
 }
 
-func (s *InsecureServingInfo) NewLoopbackClientConfig(token string) (*rest.Config, error) {
```

Comment:
@sttts Instead of removing `token` parameter here, should we pass token to the client and put it into HTTP headers for bearer authentication?

Reply:
This is the insecure client which has no authn/authz anyway. No token needed. The param only existed for symmetry of this func with the SecureServing case.